src/components/App/styles.module.scss

```diff
@@ -2,10 +2,12 @@
   text-align: center;
 }
 
-.header {
-  align-items: center;
+.content {
   background-color: #282c34;
   flex-direction: column;
-  justify-content: center;
+  padding-top: 20px;
   min-height: 100vh;
 }
+
+.login-message {
```

Comment:
Why is there an empty style for this?

Reply:
I needed a CSS class for a test case, and because we're using css modules, I decided to use this feature rather than adding a "string" class name. Does that make sense?

Reply:
Tbh, I'm not sure that it does. It seems a bit weird to me, but if you're happy with it I'm not going to argue.

Reply:
I'm just a bit worried that we might end up with a lot of cases like this, causing unnecessary bloat and distraction to our css files. But I understand your desire not to mix string classNames with module classNames.

Reply:
My main concerns are: consistency (css modules vs string class names) and being future-proof (if we need to add some style, we'll need to rely on the css module).

Reply:
Fair enough 👍